tutorials/polkadot-sdk/build-a-parachain/prepare-relay-chain.md

```diff
@@ -0,0 +1,210 @@
+    ```
+
+If command-line help is displayed, the node is ready to configure.
+
+## Relay Chain Configuration
+
+Every Substrate-based chain requires a [chain specification](https://docs.substrate.io/build/chain-spec/){target=\_blank}. The relay chain network’s chain specification provides the same configuration settings as the chain specification does for other networks. Many of the chain specification file settings are critical for network operations. For example, the chain specification identifies peers participating in the network, keys for validators, boot node addresses, and other information.
+
+### Sample Chain Configuration
+
+The local relay chain uses a sample chain specification file with two validator relay chain nodes—Alice and Bob—as authorities for this tutorial. Because a relay chain must have at least one more validator node running than the total number of connected parachain collators, you can only use the chain specification from this tutorial for a local relay chain network with a single parachain.
+
+If you wanted to connect two parachains with a single collator each, you would need to run three or more relay chain validator nodes. You would need to modify the chain specification and hard-code additional validators to set up a local test network for two or more parachains.
+
+### Plain and Raw Chain Specification
+
+The chain specification file is available in two formats: a JSON file in plain text and a JSON file in SCALE-encoded raw format.
+
+You can read and edit the plain text version of chain specification file. However, the chain specification file must be converted to the SCALE-encoded raw format before you can use it to start a node. For information about converting a chain specification to use the raw format, see [Customize a Chain Specification](https://docs.substrate.io/reference/how-to-guides/basics/customize-a-chain-specification/){target=\_blank}.
+
+The sample chain specification is only valid for a single parachain with two validator nodes. If you add other validators, add additional parachains to your relay chain, or want to use custom account keys instead of the predefined account, you'll need to create a custom chain specification file.
+
+Suppose you are completing this tutorial simultaneously as anyone on the same local network. In that case, you must download and modify the plain sample relay chain spec to prevent accidentally peering with their nodes. Find the following line in the plain chain spec and add characters to make the `protocolId` field unique:
+
+```json
+"protocolId": "dot",
+```
+
+## Start the Relay Chain Node
+
+Before starting block production for a parachain, you need to start a relay chain for them to connect.
+
+To start the validator nodes, follow these steps:
+
+1. Generate the chain specification file in the plain text format and use it to create the raw chain specification file. Save the raw chain specification file in a local working directory.
```

Comment:
Maybe add a note that mentions that by just generating the chain spec, your network will use the default values. That you should modify it for production chains